src/kvirc/ui/KviConsoleWindow.cpp

```diff
@@ -396,7 +393,11 @@ void KviConsoleWindow::saveProperties(KviConfigurationFile * cfg)
 
 void KviConsoleWindow::getBaseLogFileName(QString & buffer)
 {
-	buffer = QString("CONSOLE_%1").arg(context()->id());
```

Comment:
I still need a cup of coffee or 5, but Im not sure that replacing the ID number is a good thing especially since the ID is attached to each context in every other instance that is not log related..

![capture](https://cloud.githubusercontent.com/assets/3521959/16706933/f6a13d28-45b6-11e6-85f3-f370cc0ef595.PNG)

What happens if a specific console ID is reused, see comment in https://github.com/kvirc/KVIrc/issues/1710#issuecomment-167148440
And this still affected by having ```boolShowMetworkNameForConsoleWindowListEntry``` on or off?

is it possible to do this in the same format as channels etc? i.e.
>##### console_ID_YEAR **on** NETWORKNAME